data/continuously_deployed_apps.yml

```diff
@@ -4,6 +4,7 @@
 - cache-clearing-service
 - collections
 - collections-publisher
+- contacts-admin
```

Comment:
I think this might be "contacts" rather than contacts-admin. As you the slug in release is that: https://release.publishing.service.gov.uk/applications/contacts

As an aside, do you know why this is in a YAML file rather than in the DB? Seems odd we do it with a code change when it could just be a UI field as per other aspects of an application.

Reply:
If I remember correctly the reasoning was that it would only be fairly temporary, as once we migrate everything to CD there'll be no reason to display the status. A yaml file would be easier to revert than a database migration.